FIX - Test for inconsistent xref for catalog.

carlwilson opened the pull request:
- added test for inconsistent object number from xref table;
- added constant for inconsistent catalog number; and
- unit tests and resources for Doc Catalog.
Closes #208

carlwilson committed:
FIX - Test for inconsistent xref for catalog
- added test for inconsistent object number from xref table;
- added constant for inconsistent catalog number; and
- unit tests and resources for Doc Catalog.
Closes #208

david-russo commented on jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/PdfModule.java:
```diff
@@ -199,6 +196,7 @@
 	private static final String DICT_KEY_LANG = "Lang";
 	private static final String DICT_KEY_PAGES = "Pages";
 	private static final String DICT_KEY_PAGE_LABELS = "PageLabels";
+    private static final String DICT_KEY_TYPE = "Type";
```
Mixed indentation white-space.

david-russo commented on jhove-modules/src/test/java/edu/harvard/hul/ois/jhove/module/pdf/PdfModuleTest.java:
```diff
@@ -0,0 +1,118 @@
+/**
+ * 
+ */
```
Empty Javadoc.

david-russo commented on jhove-modules/src/test/java/edu/harvard/hul/ois/jhove/module/pdf/TestUtils.java:
```diff
@@ -0,0 +1,95 @@
+/**
+ * 
+ */
```
Empty Javadoc.

david-russo commented on jhove-modules/src/test/java/edu/harvard/hul/ois/jhove/module/pdf/TestUtils.java:
```diff
@@ -0,0 +1,95 @@
+
+	/**
+	 * 
+	 */
```
Empty Javadoc.

david-russo commented on jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/PdfModule.java:
```diff
@@ -1655,14 +1651,41 @@ protected boolean readDocCatalogDict(RepInfo info)
+                    // If the type key value is not Catalog
+                    info.setWellFormed(false);
+                    info.setMessage(new ErrorMessage(MessageConstants.ERR_DOC_CAT_TYPE_NO_CAT, 0));
+                    info.setMessage(new ErrorMessage(MessageConstants.ERR_DOC_CAT_TYPE_NO_CAT, 0));
```
Duplicate messages?

carlwilson committed:
REVIEW - Addressed review comments.
- removed empty JavaDoc;
- fixed inconsistent indentation; and
- removed duplicate error message.

carlwilson committed:
Merge branch 'integration' into fix/pdf-xref-consist
